tutorials/deploy-laravel-on-serverless-containers/index.mdx

```diff
@@ -0,0 +1,354 @@
+
+    ```
+    # stubs/nginx/http.d/default.conf
+    server {
+        listen 80;
+        listen [::]:80;
+        server_name _;
+        root /var/www/html/public;
+    
+        add_header X-Frame-Options "SAMEORIGIN";
+        add_header X-Content-Type-Options "nosniff";
+    
+        index index.php;
+    
+        charset utf-8;
+    
+        location / {
+            try_files $uri $uri/ /index.php?$query_string;
+        }
+    
+        location = /favicon.ico { access_log off; log_not_found off; }
+        location = /robots.txt  { access_log off; log_not_found off; }
+    
+        error_page 404 /index.php;
+    
+        location ~ \.php$ {
+            fastcgi_pass unix:/var/run/php/php8.2-fpm.sock;
+            fastcgi_param SCRIPT_FILENAME $realpath_root$fastcgi_script_name;
+            include fastcgi_params;
+        }
+    
+        location ~ /\.(?!well-known).* {
+            deny all;
+        }
+    }
+    ```
+
+    ```
+    # stubs/nginx/nginx.conf
+    error_log /var/log/nginx/error.log notice;
+    events {
+        worker_connections 1024;
+    }
+    http {
+        default_type application/octet-stream;
+        gzip on;
+        include /etc/nginx/mime.types;
+        include /etc/nginx/http.d/*.conf;
+        keepalive_timeout 65;
+        log_format main '$remote_addr - $remote_user [$time_local "$request" ' '$status $body_bytes_sent "$http_referer" ' '"$http_user_agent" "$http_x_forwarded_for"';
+        access_log /var/log/nginx/access.log  main;
+        sendfile on;
+    }
+    pid /var/run/nginx.pid;
+    user nginx;
+    worker_processes auto;
+    ``` 
+
+4. Create the php-fpm configuration file: the configuration file `stubs/php/php-fpm.d/zz-docker.conf` should be created to configure the php-fpm pool to render the dynamic pages of the Laravel application. Depending on the need of your application, you might have to fine-tune the configuration of the process manager. Further information is available in the [php manual](https://www.php.net/manual/en/install.fpm.configuration.php).
+    
+    ``` 
+    # stubs/php/php-fpm.d/zz-docker.conf
+    [global]
+    daemonize = no
+
+    [www]
+    listen = /var/run/php/php8.2-fpm.sock
+    listen.owner = www-data
+    listen.group = www-data
+    listen.mode = 0660
+
+    pm = dynamic 
+    pm.max_children = 75 
+    pm.start_servers = 10 
+    pm.min_spare_servers = 5 
+    pm.max_spare_servers = 20 
+    pm.process_idle_timeout = 10s
+    ```
+
+5. Build the docker image: we run the following command in our terminal.
+    ```
+    docker build -t my-image .
+    ```
+
+## Creating Container Registry 
+
+1. Create a Container Registry: we use the Scaleway console to create a container registry within the `PAR` region. We don't want our container to be retrieved without proper authentication and authorisation, so we set the visibility to `Private`.  [Create a Container Registry](https://console.scaleway.com/registry/namespaces)
```

Comment:
```suggestion
1. [Create a  ScalewayContainer Registry namespace](/containers/container-registry/how-to/create-namespace/) in the `PAR` region. Set the visibility to `Private` to avoid having your container retrieved without proper authentication and authorization.
```